manifests/image-references

```diff
@@ -5,4 +5,56 @@ spec:
   - name: cluster-monitoring-operator
     from:
       kind: DockerImage
-      name: docker.io/openshift/origin-cluster-monitoring-operator:latest
+      name: quay.io/openshift/origin-cluster-monitoring-operator:latest
+  - name: prometheus-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-prometheus-operator:latest
+  - name: prometheus-config-reloader
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-prometheus-config-reloader:latest
+  - name: configmap-reload
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-configmap-reload:latest
+  - name: prometheus
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-prometheus:latest
+  - name: prometheus-alertmanager
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-prometheus-alertmanager:latest
+  - name: grafana
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/grafana:latest
+  - name: oauth-proxy
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/oauth-proxy:latest
+  - name: prometheus-node-exporter
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-prometheus-node-exporter:latest
+  - name: kube-state-metrics
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-kube-state-metrics:latest
+  - name: kube-rbac-proxy
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-kube-rbac-proxy:latest
+  - name: telemeter
```

Comment:
here we call it `telemeter`, above we call it `telemeter-client`, those don't have to be in sync?

Reply:
this has to be consistent with the prow job so it must be telemeter